bin/import_cv2.py

```diff
@@ -53,23 +53,25 @@ def _preprocess_data(audio_dir, tsv_dir):
 def _maybe_convert_set(audio_dir, input_tsv):
     output_csv =  path.join(audio_dir,os.path.split(input_tsv)[-1].replace('tsv', 'csv'))
     print("Saving new DeepSpeech-formatted CSV file to: ", output_csv)
-    
+
     # Get audiofile path and transcript for each sentence in tsv
     samples = []
     with open(input_tsv) as input_tsv_file:
         reader = csv.DictReader(input_tsv_file, delimiter='\t')
         for row in reader:
             samples.append((row['path'], row['sentence']))
-            
+
     # Keep track of how many samples are good vs. problematic
     counter = { 'all': 0, 'too_short': 0, 'too_long': 0 }
     lock = RLock()
     num_samples = len(samples)
     rows = []
-    
+
     def one_sample(sample):
         """ Take a audio file, and optionally convert it to 16kHz WAV """
         mp3_filename = path.join(audio_dir, sample[0])
+        if not mp3_filename.lower().endswith('.mp3'):
```

Comment:
nit: I think `path.splitext(mp3_filename.lower())[0] == '.mp3'` would be better :)

Reply:
You mean `path.splitext(mp3_filename.lower())[1] == '.mp3'` nop :) ?
Done.